packages/ui/src/context/util/requestMediaAccess.ts

```diff
@@ -0,0 +1,23 @@
+import log from "loglevel"
+
+/**
+ * User gesture to request access to video
+ * @returns boolean if user granted permissions to video
+ */
+export const requestMediaAccess = async () => {
+    try {
+        const mediaAccess = await navigator.mediaDevices.getUserMedia({
+            video: true,
+            audio: false,
+        })
+
+        if (mediaAccess.getVideoTracks().length === 0) {
+            log.error("Can't access webcam with extension")
```

Comment:
do we handle the case where there is no webcam available or if it's not working properly? I didn't see it in our testing session. I think we should try to display a message if possible

Reply:
@MartnLecam

Reply:
![Message when selecting Keystone](https://user-images.githubusercontent.com/106965328/223875573-af491548-f917-482a-8e4a-39fffe72520c.png)
![Missing Camera Connection](https://user-images.githubusercontent.com/106965328/223875594-fdfade3b-68a7-4b3d-b287-453656a24cf8.png)
![Missing Camera TX](https://user-images.githubusercontent.com/106965328/223875606-42569a48-b118-4764-807f-fcbfe3d924d3.png)